pages/common/ffe.md

```diff
@@ -0,0 +1,29 @@
+# ffe
+
+> Extract fields from a flat database file and write to another format.
+> `ffe` needs a text config file to interpret the input and format the output.
```

Comment:
This line isn't needed IMO, the fact very command has a config file used, kind of gives it away

Reply:
Disagree. ‘ffe” can also use a default ~/fferc config with a collection of structures and formats that you commonly use and select from. The point is that it needs _some_ config file and there is no avoiding writing one to describe your data. I will remove -c from the -s and -f examples and mention ~/fferc there to illustrate this mode better.

Reply:
What about this?
```suggestion
> A configuration file is required to interpret the input and format the output.
```

Reply:
I incorporated this.